src/plugins/flattenSetter/flattenSetter.ts

```diff
@@ -362,12 +362,12 @@ export async function processRequest(host: Host): Promise<void> {
     else {
         Helper.dumper.dumpCodeModel("flatten-set-pre");
 
-        const m4FlattenModels = await session.getValue('modelerfour.flatten-models', false);
-        if (m4FlattenModels !== true)
-            Helper.logWarning('modelerfour.flatten-models is not turned on');
-        const m4FlattenPayloads = await session.getValue('modelerfour.flatten-payloads', false);
-        if (m4FlattenPayloads !== true)
-            Helper.logWarning('modelerfour.flatten-payloads is not turned on');
+        // const m4FlattenModels = await session.getValue('modelerfour.flatten-models', false);
+        // if (m4FlattenModels !== true)
+        //     Helper.logWarning('modelerfour.flatten-models is not turned on');
+        // const m4FlattenPayloads = await session.getValue('modelerfour.flatten-payloads', false);
+        // if (m4FlattenPayloads !== true)
+        //     Helper.logWarning('modelerfour.flatten-payloads is not turned on');
```

Comment:
why comment?

Reply:
These 2 parameters are not used in this plugin, so I think print log may cause confusing.

Reply:
then please remove it.

---
In reply to: [448990870](https://github.com/Azure/autorest.clicommon/pull/79#discussion_r448990870) [](ancestors = 448990870)